README.md

```diff
@@ -36,6 +36,7 @@ pip install m2cgen
 - Python
 - R
 - Visual Basic
+- Dart
```

Comment:
Please save alphabetical order here. It will help users find (or not find 😄 ) their target language faster.

Reply:
Shoot. Yes! My apologies.